Comparison shopping engine/Forms/SearchForm.cs

```diff
@@ -154,15 +152,18 @@ private int CountHowMany<T>(T value)
         {
             if (value.GetType() == typeof(MainGroups))
             {
-                return Product.productList.Count(product => product.Group.Contains(value.ToString()));
+                return _items.Count(product => product.Group.Contains(value.ToString()));
             }
 
-            return Product.productList.Count(product => product.Source.Contains(value.ToString()));
+            return _items.Count(product => product.Source.Contains(value.ToString()));
 
         }
         private void SearchForm_Shown(object sender, EventArgs e)
```

Comment:
Gal vietoj to, kad viską užkraudinėtų metode kuomet langas jau pasijungia reiktų visą tai daryti kuomet langas kraunasi?
Turiu omenyje perkelti iš SearcForm_Shown metodo į load metodą

Reply:
Gera įdėja, tiesiog perkelsiu metodus.